### PR TITLE
Nested expression fix

### PR DESCRIPTION
**Pull request by yesamer**
@tiagobento 

https://user-images.githubusercontent.com/16005046/228558481-cf1b2adb-caf6-4ff5-aa97-fcb017234310.mov

### CODE REVIEW

File: packages/boxed-expression-component/showcase/index.tsx [hunk 1]
```diff
@@ -94,10 +94,10 @@ const INITIAL_EXPRESSION: ExpressionDefinition = {
 
 //Defining global function that will be available in the Window namespace and used by the BoxedExpressionEditor component
 const beeGwtService: BeeGwtService = {
-  getDefaultExpressionDefinition(logicType: string): ExpressionDefinition {
+  getDefaultExpressionDefinition(logicType: string, dataType: string): ExpressionDefinition {
     return getDefaultExpressionDefinitionByLogicType(
       logicType as ExpressionDefinitionLogicType,
-      { dataType: DmnBuiltInDataType.Undefined } as ExpressionDefinition,
+      dataType as ExpressionDefinition,
```
Comment (tiagobento) [inline packages/boxed-expression-component/showcase/index.tsx:100]: Just this casting here that I think is not right?

Comment (tiagobento) [reply packages/boxed-expression-component/showcase/index.tsx:100]: I mean, that second argument should be either a string or a DmnBuiltInDataType, right?

**Review by tiagobento — CHANGES_REQUESTED** [review]
AWESOME!!!! 

Just for clarity, can you please take a look at the inline comment I added?

File: packages/boxed-expression-component/showcase/index.tsx (hunk 1 above)
Comment (yesamer) [reply packages/boxed-expression-component/showcase/index.tsx]: Done! :)

**Pull request merged**